examples/sinatra/views/saml_post.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  </head>
```

Comment:
why isn't the html header required in the `login.erb` view but you need it here?